src/App.js

```diff
@@ -0,0 +1,80 @@
+import { useState } from "react";
+
+function Square({ value, onSquareClick }) {
+  return (
+    <button className="square" onClick={onSquareClick}>
+      {value}
+    </button>
+  );
+}
+
+export default function Board() {
+  const [xIsNext, setXIsNext] = useState(true);
+  const [squares, setSquares] = useState(Array(9).fill(null));
+
+  function handleClick(i) {
+    if (squares[i] || calculateWinner(squares)) {
+      return;
+    }
+    const nextSquares = squares.slice();
+
+    if (xIsNext) {
+      nextSquares[i] = "X";
+    } else {
+      nextSquares[i] = "O";
+    }
+    setSquares(nextSquares);
+    setXIsNext(!xIsNext);
+  }
+
+  const winner = calculateWinner(squares);
+  let status;
+  if (winner) {
+    status = winner + "'s wins this game!";
+  } else {
+    status = "Next player is " + (xIsNext ? "X" : "O")
+  }
```

Comment:
you can avoid if/else here
```suggestion
  let status = winner
    ? winner + "'s wins this game!"
    : "Next player is " + (xIsNext ? "X" : "O")
```